### PR TITLE
feat(hooks): add react instantsearch hooks template

### PR DESCRIPTION
**Pull request by dhayab**
This PR adds a template to quickly bootstrap React InstantSearch Hooks-based applications.

The stack uses Parcel and TypeScript.

### CODE REVIEW

File: src/templates/React InstantSearch Hooks/src/App.tsx.hbs [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+          </a>
+        </p>
+      </header>
+
+      <div className="container">
+        <InstantSearch searchClient={searchClient} indexName="{{indexName}}">
+          <div className="search-panel">
+            <div className="search-panel__filters">
+              {{#if flags.dynamicWidgets}}
+              <DynamicWidgets>
```
Comment (Haroenv) [inline src/templates/React InstantSearch Hooks/src/App.tsx.hbs:47]:
```suggestion
              <DynamicWidgets fallbackWidget={RefinementList}>
```

File: src/templates/React InstantSearch Hooks/src/App.tsx.hbs [hunk 2]
```diff
@@ -0,0 +1,99 @@
+import algoliasearch from 'algoliasearch/lite';
+import {
+  {{#if flags.dynamicWidgets}}
```
Comment (Haroenv) [inline src/templates/React InstantSearch Hooks/src/App.tsx.hbs:3]: these imports can be like https://github.com/algolia/create-instantsearch-app/blob/d9358cde3b212fe29d5f35d47c3a0de7250d9d50/src/templates/React%20InstantSearch/src/App.js.hbs#L7-L19 with fallbackWidget

**Review by Haroenv — COMMENTED** [review]
👍

**Review by Haroenv — APPROVED** [review]

**Comment by Haroenv** [comment]
looks like you need to update the tests still :)

**Review by FabienMotte — APPROVED** [review]

**Pull request merged**